docs/concepts/gateway/customization.md

```diff
@@ -224,6 +224,37 @@ class MyGateway(FastAPIBaseGateway):
         return app
 ```
 
+(executor-streamer)=
+## Calling an Individual Executor
+An `executor` object is injected by Jina to your gateway class which allows you to interface with individual Executors from the Gateway.
+
+After transforming requests that arrive to the gateway server into Documents, you can call the Executor in your Python code using `self.executor['executor_name'].post(args)`.
+This method expects a DocumentArray object and an endpoint exposed by the Executor (similar to Jina Client). 
+It returns a 'coroutine' which returns a DocumentArray.
+
+Here is an example of calling an individual Executor named 'executor1':
```

Comment:
I think we should give a more general example, for instance, adding a bunch of Executors to the Flow and calling one of them explicitly:
Flow.add(name='executor1', uses=...).add(name='executor2',...)...
// then show how to call one of them
Maybe we should also showcase how to call 2 executors in parallel using asyncio.

Reply:
Yea sure. Done